src/consensus/params.rs

```diff
@@ -105,6 +112,20 @@ impl Params {
                 allow_min_difficulty_blocks: true,
                 no_pow_retargeting: false,
             },
+            Network::Signet => Params {
+                network: Network::Signet,
+                bip16_time: 1333238400,                 // Apr 1 2012
+                bip34_height: 0,
+                bip65_height: 0,
+                bip66_height: 0,
+                rule_change_activation_threshold: 1916, // 95%
+                miner_confirmation_window: 2016,
+                pow_limit: MAX_BITS_SIGNET,
+                pow_target_spacing: 10 * 60,            // 10 minutes.
+                pow_target_timespan: 14 * 24 * 60 * 60, // 2 weeks.
+                allow_min_difficulty_blocks: false,
+                no_pow_retargeting: false,
```

Comment:
The above parameters can be verified here: https://github.com/kallewoof/bitcoin/blob/2b9db9ac0936cf278ad27c59497d9feec61f6440/src/chainparams.cpp#L295-L306

@kallewoof Any particular reason the `BIP{34,65,66}` heights differ from your Core implementation?

Reply:
Mistake; changed here to match (425446c).